### PR TITLE
itk-snap 4.0.2,15357

### PR DESCRIPTION
**Pull request by krehel**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/i/itk-snap.rb [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
   on_arm do
-    version "4.0.2,13696"
+    version "4.0.2,15357"
     sha256 "aba01235a44b9080fae2c5cf0b35485f96b335f1a50442ee4fb941d4c4846501"
   end
   on_intel do
-    version "4.0.2,13697"
+    version "4.0.2,15356"
     sha256 "333071f6703e6d3a8554fcc33f2a632cf3106a0bdc230893807a4c288e9e13da"
   end
```
[the remark this answers is not in the document]
Comment (bevanjkay) [reply Casks/i/itk-snap.rb:12]: The second part of the version is not used in the url, so the artifact did not change.

The update is superfluous, so maybe the livecheck should be adjusted instead

[the remark this answers is not in the document]
Comment (krehel) [reply Casks/i/itk-snap.rb:12]: Yeah, I can tweak the livecheck.  If the second part isn't even used in the version scheme, perhaps we don't need it at all.

[the remark this answers is not in the document]
Comment (krehel) [reply Casks/i/itk-snap.rb:12]: @samford - sorry for pulling you in here but realize you recently worked on this.  Can we drop the second capture group from the version number as that doesn't appear to be used?  Was there another reason for having it?

Thanks!

[the remark this answers is not in the document]
Comment (samford) [reply Casks/i/itk-snap.rb:12]: I was curious as to whether the `link` number (the second part of the `version`) on the first-party download page would change independently of a new stable release and lead to an unnecessary version bump PR like this, so thanks for the mention.

I think I was a little tired when reviewing the recent `itk-snap` PR, as I was focused on getting the proposed `livecheck` block to work and evidently overlooked that the cask uses a dmg file from SourceForge. The previous `livecheck` block URL redirected to the SourceForge file but the download URL on the first-party download page is something different. It directs to an optional registration page before giving a download link like http://www.itksnap.org/download/snap/13696 but that URL returns a 404 (Not Found) error.

The page after the optional registration page also contains a message below the download link that says "If the link above does not work, you can find all ITK-SNAP binary downloads in the [SourceForge files area](https://sourceforge.net/projects/itk-snap/files/itk-snap/)". This seems to suggest that checking SourceForge for new versions is fine and there may not be any benefit to checking the first-party download page (it wasn't clear to me before).

We would normally check SourceForge for new versions in this scenario anyway, so I think that's the way to go. I opened https://github.com/Homebrew/homebrew-cask/pull/166435 to update the `version` format and `livecheck` block accordingly.

**Pull request closed without merging**